Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1225,6 +1225,11 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
     return roundedIntPoint(transformedPoint);
 }
 
+WebCore::FloatRect UnifiedPDFPlugin::convertFromPageToContents(const WebCore::FloatRect& pageSpaceBounds, PDFDocumentLayout::PageIndex pageIndex) const
+{
+    return { convertFromPageToContents(WebCore::flooredIntPoint(pageSpaceBounds.location()), pageIndex), pageSpaceBounds.size().scaled(scaleFactor() * m_documentLayout.scale()) };
```

Comment:
Is this right in all rotations? I would have expected it to involve documentSpaceToPageSpaceTransform.